Tests/AuthTests/Mocks/Mocks.swift

```diff
@@ -75,7 +77,7 @@ extension Session {
     accessToken: "accesstoken",
     tokenType: "bearer",
     expiresIn: 60,
-    expiresAt: Date().addingTimeInterval(60).timeIntervalSince1970,
+    expiresAt: Date().addingTimeInterval(-60).timeIntervalSince1970,
```

Comment:
Nice catch

Reply:
/nit consider adjusting `expiresIn` to -60 or 0 since we compute issuedAt from that